src/users/users.controller.ts

```diff
@@ -22,16 +22,18 @@ export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
   @Get()
+  @UseGuards(JwtAuthGuard)
```

Comment:
if all method in class should check JWT, put JWT guard in controller

Reply:
done